keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+// place overrides here
+#define TAPPING_TERM 200
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This setting should be removed, as it's no longer used.

Specifically, it's the new default behavior for QMK Firmware.  And you can disable it by defining `STRICT_LAYER_RELEASE` instead. 

```suggestion
```